app/directives/accessControlForm/porAccessControlFormController.js

```diff
@@ -58,10 +58,13 @@ function ($q, UserService, Notifications, Authentication, ResourceControlService
     })
     .then(function success(data) {
       if (data) {
+        ctrl.formData.AccessControlEnabled = true;
```

Comment:
Oh yeah :/
I did that cause when you edit a container with public access, the form default to "Admins Only" (access control enable instead of disable)